src/test/java/net/minecraftforge/debug/FluidAdditionalFieldsTest.java

```diff
@@ -0,0 +1,81 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fluids.BlockFluidBase;
+import net.minecraftforge.fluids.BlockFluidClassic;
+import net.minecraftforge.fluids.Fluid;
+import net.minecraftforge.fluids.FluidRegistry;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventBusSubscriber;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry.ObjectHolder;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+@Mod(modid = FluidAdditionalFieldsTest.MODID, name = "Test Mod", version = "1.0.0", acceptedMinecraftVersions = "*")
+@EventBusSubscriber
+public class FluidAdditionalFieldsTest
+{
+    static final boolean ENABLED = false; // <-- enable mod
+    static final int COLOR = 0xFFd742f4; // <-- change value for testing
+
+    static final String MODID = "fluidadditionalfields";
+    static final ResourceLocation RES_LOC = new ResourceLocation(MODID, "slime");
+    static
+    {
+        if (ENABLED)
+        {
+            FluidRegistry.enableUniversalBucket();
+        }
+    }
+    public static final Fluid SLIME = new Fluid("slime", new ResourceLocation(MODID, "slime_still"), new ResourceLocation(MODID, "slime_flow")).setColor(COLOR);
+    @ObjectHolder("slime")
+    public static final BlockFluidBase SLIME_BLOCK = null;
+
+    @EventHandler
+    public void ev(FMLPreInitializationEvent event)
```

Comment:
please use descriptive method names. `preInit` is a pretty standard name for this

Reply:
okay, for event handler files, as you know, they're never called by anything so meaningful name doesn't matter, it is the parameter type that creates the meaning. However, easy change so i'll do it.

Reply:
While the method signature tells you *when* the method is called, it doesn't specify its *purpose*. Assume you're using one of the rendering events for a very specialized HUD element or something. You wouldn't want to call your method just `onRender` (as that's to be inferred from the signature), but instead give it a name that describes *what* you render.
Additionally, at least for me it's was easier to quickly parse the method name as that's not directly surrounded by other tokens, while the signature is embedded in between method and parameter name. Also, `preInit` is easily identifiable, while extracting that info from the clutter in `FMLPreInitializationEvent` is harder and might cause confusion with the non-Pre events at a quick glance.